projects/ui/src/lib/services/po-language/po-language.service.ts

```diff
@@ -25,7 +23,14 @@ const poLocaleKey = 'PO_USER_LOCALE';
   providedIn: 'root'
 })
 export class PoLanguageService {
-  constructor() {}
+  constructor() {
+    const keepLanguage = localStorage.getItem(poKeepLanguage) === 'true';
+    if (!keepLanguage) {
+      localStorage.removeItem(poDefaultLanguage);
+      localStorage.removeItem(poLocaleKey);
+    }
+    localStorage.removeItem(poKeepLanguage);
```

Comment:
o próximo refresh apagaria as configurações mesmo que essa não seja a ação padrão que ocorria

Reply:
Ele sempre apaga na carga ou recarga, exceto forçada pelo `setLanguage(idioma, true)`. Na próxima vez que o usuário entrar a aplicação será apagado, isso ainda é um problema? Se sim estou pensando em reverter!

Reply:
Sim, não é o comportamento que ocorria e também não me parece correto considerar apagar por padrão.

Reply:
Como sugestão abri um PR com uma solução via parâmetro ao configurar o module: https://github.com/po-ui/po-angular/pull/1462